### DISCUSSION TITLE
Change scans result dir

### DISCUSSION THREAD

**Discussion by Shelestov7** [cea-sec/ivre#1093] [Q&A] [answered]
Hello can i change dir where saves xml files when i run ivre runscans --output=XMLFork

**Comment by p-l-** [comment, marked as answer]
Hi,

That is not possible. The only way to do that would be to `cd` to the directory where you want your results stored before running `ivre runscans`.

**Reply by Shelestov7** [reply]
Tnx for answer and your app.
